core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -127,6 +127,11 @@ object TimeSeriesShard {
   val indexTimeBucketTtlPaddingSeconds = 24.hours.toSeconds.toInt
   val indexTimeBucketSegmentSize = 1024 * 1024 // 1MB
 
+  // Flush groups when ingestion time is observed to cross an hourly boundary. This simplifies
+  // disaster recovery -- chunks can be copied without concern that they may overlap in time.
+  // TODO make configurable if necessary
+  val flushBoundaryMillis = 1.hour.toMillis
```

Comment:
See `StoreConfig`... there is a configuration already for the flush interval, we should just use that.

Reply:
ok